server/flyway/sql/V54__add_role_display_name.sql

```diff
@@ -0,0 +1,112 @@
+-- Add role display_name
+ALTER TABLE app_fam.fam_role ADD COLUMN display_name VARCHAR(100);
+
+COMMENT ON COLUMN app_fam.fam_role.display_name IS 'Human redable name for the role';
+
+-- Migrate existing roles to set display_name below; for all 'C' and 'A' role types and all child roles.
+
+-- CLIENT application roles: ('CLIENT_ADMIN%', 'CLIENT_EDITOR%', 'CLIENT_VIEWER%');
+-- 'CLIENT_ADMIN' (role_type = 'C')
+UPDATE app_fam.fam_role SET display_name = 'Admin'
+    WHERE role_id in (
+    SELECT role.role_id
+    FROM app_fam.fam_role role
+    WHERE role.role_name like 'CLIENT_ADMIN%');
```

Comment:
Hi Ian, wondering do we need "%" at the end for concrete role? thanks!

Reply:
Ha, no, but it does not matter actually; if concrete role, it will just find the concrete role id (only 1).  
I give every script the same to be easier to write.